packages/shared/src/utils/constants/contract.tsx

```diff
@@ -394,6 +394,11 @@ export const getUnsupportedContracts = () =>
         },
     } as const);
 
+/**
+ * // Config to display trade buttons and their positions
```

Comment:
@jim-deriv this is not the only purpose of this config :) e.g. it's also use to get a name of a trade type for statement in reports